### ISSUE TITLE
Incorrect resizing with latest JWL (title bar hidden)

### ISSUE THREAD

**Issue by TizianoNoto** [AntonyCorbett/JwlMediaWin#18]
I have a dual monitor setup so I still use JwlMediaWin to share pictures and videos during the meetings more easily and without accidents.

However, in JW Library 12.3.106 the media window ~no longer has~ is created with the title bar hidden. Only the (now transparent) buttons may be slightly visible in windowed mode, depending on whether the window is selected or not. This means at least two things:
   - the window now extends too much beyond the top border of the screen so the media displayed is not centered;
   - the resizing is not needed and maximizing the window should be performed in its place.

Again, thank you for the hard work you do in order to help us manage meetings as smoothly as possible!

_Edited for more clarity._

**Comment by AntonyCorbett** [comment]
@TizianoNoto Thanks for your report. I can't reproduce the problem you are experiencing and wonder if you would expound a little, please? AFAIK, the media window has never had a title bar when shown on a second monitor. Nor do I see any buttons.

**Comment by LvDaniel** [comment]
It might be related to JWL 12.3.106 or might be due to some Windows updates. I also noticed today that the JWL Media window has a black bar on the bottom of the screen. Also the brother that shared the screen during today's meeting had this issue (we're using JWL 12.3.106 and the latest JWLMediaWin 1.0.0.9)

I've test this on a different laptop where I have JWL 12.2.69 + JWLMediaWin 1.0.09 and everything runs smoothly there.

![JWL Media Screen](https://user-images.githubusercontent.com/45002147/104845443-3760b200-58de-11eb-911a-1106d3a601bc.png)

**Comment by TizianoNoto** [comment]
@AntonyCorbett of course. I'll try to explain myself better. When the second display option is enabled in JWL, it creates a new window (the "media window"), called "JW Library", in full screen mode on the second display. If no second display is detected, the media window will be opened in normal mode.

They must have changed something in JWL 12.3. The "normal" media window now doesn't show a title bar. Only the buttons can be spotted in the top right corner and will be white if window is active, otherwise they're black. See the following screenshot.
![JwlMediaWindow](https://user-images.githubusercontent.com/42154401/104846905-45fa9980-58dd-11eb-87b5-c7384aba0980.png)

My understanding is that JwlMediaWin sets the media window in normal mode, moves it on the second display and resizes it so that its content match the resolution without showing the title bar and the borders. But what if there is no title bar? The content of the window now has a portion of it beyond the top of the screen and theree is some unused space at the bottom, as it's shown in @LvDaniel 's screenshot.

My suggested fix is to see if the title bar is visible (_AppWindowTitleBar.IsVisible_ property?) and decide the appropriate behaviour (resize or just maximize).

**Comment by AntonyCorbett** [comment]
> I also noticed today that the JWL Media window has a black bar on the bottom of the screen.

Is this with or without JwlMediaWin?

**Comment by AntonyCorbett** [comment]
I can reproduce the problem (in my case it's only apparent if the aspect ratio of the monitor and image means that the image normally occupies the full height of the display).

**Comment by AntonyCorbett** [comment]
The titlebar remains but it is transparent. Will fix

**Comment by AntonyCorbett** [comment]
@TizianoNoto @LvDaniel 

I've updated the application (v1.1.0.1). I'd be grateful if you would take a look and report back please.

**Comment by LvDaniel** [comment]
> > I also noticed today that the JWL Media window has a black bar on the bottom of the screen.
> 
> Is this with or without JwlMediaWin?

It's with JwlMediaWin.
I'll test v1.1.0.1 and reply back.

**Comment by LvDaniel** [comment]
> @TizianoNoto @LvDaniel
> 
> I've updated the application (v1.1.0.1). I'd be grateful if you would take a look and report back please.

JW Media window occupies the full height of the display now.
I can see like @TizianoNoto the buttons in the top right corner, but they are not so disturbing.

![JWL Media Screen](https://user-images.githubusercontent.com/45002147/104854000-398e3500-590d-11eb-88a8-794e41ccd717.png)

**Comment by TizianoNoto** [comment]
I confirm that it looks fine, however I had to restart JWL after startin JwlMediaWin or the window wouldn't be fixed (related to #17 ?)

**Comment by LvDaniel** [comment]
> I confirm that it looks fine, however I had to restart JWL after startin JwlMediaWin or the window wouldn't be fixed (related to #17 ?)

@TizianoNoto are you still see the buttons on the top right corner or they have disappeared?
After I've upgrade to v1.1.0.1 I can see them, but they were not visible in the previous version v1.0.09

![image](https://user-images.githubusercontent.com/45002147/104858167-1754e100-5926-11eb-81cf-d2a096f58762.png)

**Comment by TizianoNoto** [comment]
> @TizianoNoto are you still see the buttons on the top right corner or they have disappeared?

Yes.

Up to JWL 12.2, the media window would have a solid title bar with the buttons (unless in UWP full screen mode) and JwlMediaWin would "crop" it. With the current version, the bar is fully transparent meaning you can't "crop" the buttons without losing a piece of the media displayed.

Why the change? Many brothers don't have two monitors (or another device) to stream the media and monitor other things at the same time without distracting the audience. JWL has been allowing for some time to enable a separate media window even if you have a single display. My guess is it was done for this scenario. Since the title bar was solid, it would appear in the stream wasting precious space (and looking more "amateur"). Having a fully transparent title bar will help those brothers.

**Comment by LvDaniel** [comment]
@TizianoNoto thanks for info.
I noticed that if I don't use JwlMediaWin those buttons (in the top right) are not present. I always use a second screen to display JWL content so I'm not using a separate JWL window on the same screen.

@AntonyCorbett is it possible to make disappear those buttons or this is the normal behavior?
Thank you brothers!

**Comment by AntonyCorbett** [comment]
> @AntonyCorbett is it possible to make disappear those buttons or this is the normal behavior?

@LvDaniel Possibly, but not without some digging. @TizianoNoto explains it well in his previous comment.
